### ISSUE TITLE
Tables

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1362]
**Component block(s):** table

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=197-2312&t=CW4GvhJcfl45Hv5O-0

As a Web Liaison, I want the ability to present content in a uniformly styled table so that it's easy to share information with web users.

**Requirements:** 
Update the table styling to include: 

- Heading row "Heading M" style, white text, primary background color
- Table will have left aligned text
- Table will include bottom border

Update the 'Striped' table styling to include: 

- Heading row "Heading M" style, white text, primary background color
- Table will have left aligned text
- Table will include bottom border
- Every other row has grey background color #EEEEEE

Both table variations include the following mobile functionality: 

- Mobile table includes fade on direction 
- Mobile table is scrollable
- Mobile table includes left right arrow indicators for scroll
- Left/right arrows are only active when additional content is available in that direction, display as disabled when scrolled to table edge in that direction
- Left/right arrows are clickable. On click scroll % or +1 column (add this to working session review)

Update the Table (agenda, no-header): 

- Would like to see width options available - working session with Patrick when we get to this implementation to create 2-3 width options for the appropriate agenda use cases
- Mobile agenda table should be column widths 40/15/15/15/15 (validate this with Patrick on implementation)
- Leverage design table styling for the agenda table colors, spacing, etc

**Acceptance Criteria:** 
1. Given I am authoring a page, when I add a component from the library, then I am able to add a Table component.
2. Given I am viewing a page with a table, when the page renders, then the table includes a header row with primary background color, and white "Heading M" sized text. 
3. Given I am viewing a page with a table, when the page renders, then the table has left aligned text. 
4. Given I am viewing a page with a table, when the page renders, then the table rows include a bottom border. 
5. Given I am authoring a page, when I add a component from the library, then I am able to add a Striped Table component.
6. Given I am viewing a page with a striped table, when the page renders, then the table includes a header row with primary background color, and white "Heading M" sized text. 
7. Given I am viewing a page with a striped table, when the page renders, then the table has left aligned text. 
8. Given I am viewing a page with a striped table, when the page renders, then the table rows include a bottom border. 
9. Given I am viewing a page with a striped table, when the page renders, then every other table row includes an #EEEEEE row background color (to create the 'stripe' effect).
10. Given I am viewing a table on mobile, when the table renders, then it is defaulted to the leftmost column in view.
11. Given I am viewing a table on mobile, when there is more content than fits on the screen, then the table is scrollable and a fade displays on the right side of the table to indicate scroll. 
12. Given I am viewing a table on mobile, when I want to see more table information, then I am able to swipe to scroll left/right on the table. 
13. Given I am viewing a table on mobile, when I begin to scroll, then the fade corresponds to which direction of the table has additional data to show. 
14. Given I am viewing a table on mobile, when there is additional content and the table is scrollable, then left/right arrows display below the table. 
15. Given I am viewing a table on mobile, when there is additional columns of content to scroll, then the left/right arrow displays as active.
16. Given I am viewing a table on mobile, when I have reached the end of the scrollable content in a given direction, then that arrow disables.

**Comment by sarah0arnold** [comment]
Add agenda table + width options
Agenda table mobile 40/15/15/15/15 (validate use case)
Implement working session during development to validate setup

**Comment by beccasuchanek** [comment]
**Pending Questions**

Test Page URL: https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/tables

1. Go to mobile screen size where the table needs to be scrollable horizontally and see if you can do it with your keyboard to scroll left and right because I can, so I don't think this is an issue. Asked @sarah0arnold to test to ensure not an issue.

<img width="1499" height="829" alt="Image" src="https://github.com/user-attachments/assets/7e296489-22a2-4d44-8f05-72304b1041a6" />

2. This is not horizontally aligned but only having on Table Agenda, No Header. I checked authoring for the issues, didnt see anything and don't see any CSS styling making the space in the <p>. Asked dev chat for any ideas of this issue (team bandwidth is low).

<img width="1279" height="300" alt="Image" src="https://github.com/user-attachments/assets/712a4d7c-b338-4053-b815-c89bf37d5980" />

**Comment by sarah0arnold** [comment]
Tables reviewed in UAT session with project team 8/4/25 and passes. Closing ticket
Outstanding item to be addressed for agenda table in ticket #1478
